packages/tools/generator-pie-component/package-lock.json

```diff
@@ -0,0 +1,2953 @@
+{
```

Comment:
This can be reverted, I think it was left over from when we ran the `npm link` command 👍🏼

Reply:
Sure, will remove it. Do you think it can be added in the .gitignore of generator-pie-component for when npm link is ran again in that folder in future?